views/pages/dynamic.ejs

```diff
@@ -3,15 +3,28 @@
     <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Plugin Framework QA</title>
+    <title>Simple Plugin Example</title>
     <style>
         body {
-        background-color: lightblue;
-        display: flex;
-        height: 100vh;
-        width: 100vw;
-        align-items: center;
-        justify-content: center;;
+            background-color: lightblue;
+            display: flex;
+            height: 100vh;
+            width: 100vw;
+            align-items: center;
+            justify-content: center;;
```

Comment:
While you're under the hood, can you please remove the extra semicolon at the end of line 14? Thanks!

Reply:
Good catch! Fixed in 4592b9213a61ea3c4671e57c94abc57056555f9e